Detectors/GlobalTrackingWorkflow/src/TPCITSMatchingSpec.cxx

```diff
@@ -248,15 +248,14 @@ void TPCITSMatchingDPL::run(ProcessingContext& pc)
     mMatching.setTPCTrkLabelsInp(lblTPCPtr);
   }
 
-  //  const gsl::span<o2::ft0::RecPoints> fitInfo; // not yet POD
-  std::vector<o2::ft0::RecPoints> fitInfo;
   if (o2::globaltracking::MatchITSTPCParams::Instance().runAfterBurner) {
-    //    fitInfo = pc.inputs().get<gsl::span<o2::ft0::RecPoints>>("fitInfo");
-    fitInfo = pc.inputs().get<std::vector<o2::ft0::RecPoints>>("fitInfo");
-    mMatching.setFITInfoInp(&fitInfo);
+    // Note: the particular variable will go out of scope, but the span is passed by copy to the
+    // worker and the underlying memory is valid throughout the whole computation
+    auto fitInfo = pc.inputs().get<gsl::span<o2::ft0::RecPoints>>("fitInfo");
+    mMatching.setFITInfoInp(fitInfo);
   }
 
-  /*
+  /* is this code deprecated?
```

Comment:
@shahor02 is the commented code below deprecated? Then I can delete it, otherwise I have to rearrange a bit if the code is intended to be reactivated.

Reply:
Yes, it can be deleted, thanks!

Reply:
done